### PR TITLE
Add FTs as certifier for dev

### PR DESCRIPTION
**Pull request by Spuffynism**

### CODE REVIEW

File: cypress/server/index.html [hunk 1]
```diff
@@ -11,6 +11,8 @@
         var scriptVersion = keepQueryParamIfDefined('version') || '2';
         var scriptUrl =
             keepQueryParamIfDefined('staticurl') ||
+            (keepQueryParamIfDefined('s3Path') &&
```
Comment (Spuffynism) [inline cypress/server/index.html:14]: Not sure if this fallback behaviour is what we want (staticurl -> s3 -> fallback)

File: cypress/server/index.html [hunk 2]
```diff
@@ -11,6 +11,10 @@
         var scriptVersion = keepQueryParamIfDefined('version') || '2';
         var scriptUrl =
             keepQueryParamIfDefined('staticurl') ||
+            (keepQueryParamIfDefined('targetEnv').toLowerCase() === 'dev' &&
+                'https://coveo-ndev-binaries.s3.amazonaws.com/proda/StaticCDN/coveo.analytics.js/' +
+                    scriptVersion +
+                    '/coveoua.js') ||
             'https://static.cloud.coveo.com/coveo.analytics.js/' + scriptVersion + '/coveoua.js';
```
Comment (francoislg) [inline cypress/server/index.html:18]:
```suggestion
        var urlPerEnv = {
           "dev": "https://coveo-ndev-binaries.s3.amazonaws.com/proda/StaticCDN/coveo.analytics.js/",
           "prod": "https://static.cloud.coveo.com/coveo.analytics.js/"
        };
        var env = (keepQueryParamIfDefined('targetEnv') || "dev").toLowerCase();
        var scriptUrl =
            keepQueryParamIfDefined('staticurl') || urlPerEnv[env] + scriptVersion + '/coveoua.js';
```

Even if it "wastes" a couple of variables and query param checks, I think it helps for readability

Comment (Spuffynism) [reply cypress/server/index.html:18]: Yes I agree, that's much much clearer! I have one question though: When the current environment is the production environment, the `targetEnv` that's passed to the certifier seems to have a value of `prd` and not `prod`, so should `"prod"` be `"prd"` instead?

Comment (francoislg) [reply cypress/server/index.html:18]: yeah no issue with `prd` :P

**Review by dlafreniere — APPROVED** [review]

**Review by francoislg — APPROVED** [review]

**Pull request merged**